integration/experiment/power_sweep/gen_policy_recommendation.py

```diff
@@ -149,50 +145,38 @@ def extract_columns(df, region_filter = None):
     is a container that specifies which regions to include (by default,
     include all of them)."""
     df_filtered = df
-    if region_filter:
+    if region_filter and region_filter != 'Epoch':
         df_filtered = df[df['region'].isin(region_filter.split(','))]
 
     # these are the only columns we need
-    df_cols = df_filtered[[
-        'POWER_PACKAGE_LIMIT_TOTAL',
-        'host',
-        'runtime (sec)',
-        'package-energy (joules)']] \
-    .rename({
-        'POWER_PACKAGE_LIMIT_TOTAL': 'power_limit',
-        'runtime (sec)': 'runtime',
-        'package-energy (joules)': 'energy'
-    },
-    axis = 1) \
-    .groupby(['power_limit', 'host']).sum().reset_index() \
-    [['power_limit', 'runtime', 'energy']] \
-    .set_index('power_limit')
+    df_cols = df_filtered[['POWER_PACKAGE_LIMIT_TOTAL',
+                           'host', 'Profile',
+                           'runtime (sec)',
+                           'package-energy (joules)']]
+
+    df_cols = df_cols.rename({'POWER_PACKAGE_LIMIT_TOTAL': 'power_limit',
+                              'runtime (sec)': 'runtime',
+                              'package-energy (joules)': 'energy'
+                             }, axis = 1)
+
+    df_cols = df_cols.groupby(['power_limit', 'host', 'Profile']).sum().reset_index()
+    df_cols = df_cols[['power_limit', 'runtime', 'energy']].set_index('power_limit')
 
     return df_cols
 
 
 def dump_stats_summary(df, fname):
     """
-    Write mean runtime and energy and the standard deviation of 
+    Write mean runtime and energy and the standard deviation of
     runtime and energy for each power limit in CSV format to the
     file fname."""
-    means = df \
-            .groupby(level=0) \
-            .mean() \
-            .rename(lambda x: x+"_mean", axis=1)
-    stds = df \
-            .groupby(level=0) \
-            .mean() \
-            .rename(lambda x: x+"_stddev", axis=1)
+    means = df.groupby(level=0).mean().rename(lambda x: x+"_mean", axis=1)
+    stds = df.groupby(level=0).mean().rename(lambda x: x+"_stddev", axis=1)
```

Comment:
This should have been .std() rather than .mean()! That bug was in the old code -- would you mind fixing it?